conf/openimis.conf

```diff
@@ -12,6 +12,17 @@ server {
     ssl_certificate_key /etc/letsencrypt/live/${NEW_OPENIMIS_HOST}/privkey.pem;
     root /usr/share/nginx/html;
 
+    ssl_protocols TLSv1.2 TLSv1.3;
```

Comment:
Should we keep this change? AI suggested to add this too as a part of DH param change @delcroip